README.md

```diff
@@ -1,27 +1,57 @@
 # Axon
 
+[![version](https://img.shields.io/github/v/release/axonweb3/axon?sort=semver)](https://github.com/axonweb3/axon/releases) [![license](https://img.shields.io/github/license/axonweb3/axon)](https://github.com/axonweb3/axon/blob/main/LICENSE) ![status-check](https://github.com/axonweb3/axon/actions/workflows/web3_compatible.yml/badge.svg?branch=main)  ![contributors](https://img.shields.io/github/contributors/axonweb3/axon)
+
 > Axon is still in active development and a work in progress.
 
-## About Axon
+Axon is a high-performance layer 2 framework with native cross-chain function.  Built on top of the [Overlord](https://github.com/nervosnetwork/overlord) consensus protocol and the P2P network [Tentacle](https://github.com/nervosnetwork/tentacle), Axon supports hundreds of nodes and achieves thousands of TPS. Axon is also EVM-compatible with well-developed toolchains. It performs high interoperability, meaning deployed applications can achieve convenient cross-chain communication.
```

Comment:
Axon is a high-performance layer 2 framework with native cross-chain function.  Built on top of the [Overlord](https://github.com/nervosnetwork/overlord) consensus protocol and the P2P network [Tentacle](https://github.com/nervosnetwork/tentacle), Axon supports hundreds of nodes and achieves thousands of TPS. Axon is also EVM-compatible with well-developed toolchains. Its high interoperability facilitates cross-chain communication among dApps.